### ISSUE TITLE
AoT support

### ISSUE THREAD

**Issue by pedropapa** [Gbuomprisco/ngx-content-loading#2]
Currently the module is not working with AoT (Which comes with Angular 5.x). While trying to build the project for prod environment I receive the following error:

```
Error during template compile of 'NgxContentLoadingModule' Function calls are not supported in decorators in
            'NgModule' 'NgModule' calls a function at @angular/core/core.ts(194,31).
```

`ng build --prod --aot=false` is a workaround for now.

**Comment by Gbuomprisco** [comment]
Seems related to https://github.com/dherges/ng-packagr/issues/607

**Comment by Gbuomprisco** [comment]
I asked for help in the thread above, hopefully they'll know what's up with this error :)

**Comment by pedropapa** [comment]
Just to inform that this bugs is fixed in the latest release of the project (0.0.9): https://github.com/Gbuomprisco/ngx-content-loading/commit/4616c521bd3d68f360f69acbed25d1f6c8803067

**Comment by Gbuomprisco** [comment]
To those interested, I was able to make it work with 0.1.0

**Comment by kapiladiyecha** [comment]
Hi @Gbuomprisco , Good to hear that, but #3 `Can not read property 'speed' of undefined` can it be fixed with 0.1.0?

Thanks
